### PR TITLE
Ref Guide - User Account

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10592

### CODE REVIEW

**Comment by Loquacity** [comment]
Tests are failing because of new section created in https://github.com/uyuni-project/uyuni-docs/pull/96 but not yet merged.

**Comment by Loquacity** [comment]
Also: PR # :100: ! :tada:

File: modules/reference/pages/home/user-account-change-email.adoc [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-Set the email {productname} sends notifications to on the menu:Main Menu[Home > User Account > Change Email] page.
-If you would like to receive email notifications about patch alerts or daily summaries for your systems, check the btn:[Receive email notifications] checkbox located on the [guimenu]``My Preferences`` page.
+The menu:Home[User Account > Account Deactivation] section allows you to set the email {productname} sends notifications to.
 
-image::home_user_account_change_email.png[scaledwidth=80%]
-
-To change your preferred email address, click menu:Main Menu[Home > User Account > Change Email].
 Enter your new email address and click the btn:[Update] button.
-Invalid email addresses like those ending in `@localhost` are filtered and rejected.
+Invalid email addresses, including those those ending in ``@localhost`` are filtered and rejected.
```
Comment (keichwa) [inline modules/reference/pages/home/user-account-change-email.adoc]:
```suggestion
Invalid email addresses, including those ending in ``@localhost`` are filtered and rejected.
```

File: modules/reference/pages/home/user-account-my-account.adoc [hunk 2]
```diff
@@ -1,17 +1,10 @@
 [[ref.webui.overview.account]]
 = My Account
 
+The menu:Home[User Account > My Account] section allows you to change user account preferences.
+
 Modify your personal information, such as name, password, and title from the menu:Main Menu[Home > User Account > My Account] page.
```
Comment (keichwa) [inline modules/reference/pages/home/user-account-my-account.adoc]:
```suggestion
Modify your personal information, such as name, password, and title from the menu:Home[User Account > My Account] page.
```

**Pull request merged**